### PR TITLE
RUNO: Migrate Sequelize to Typeorm

### PR DESCRIPTION
**Pull request by Runo-saduwa**
#### What does this PR achieve?
The PR migrates the project from sequelize to Typeorm and it consists of the following actions:
- Installs typeorm, typeorm/nest and pg
- Imports ormconfig into app module and adds it to imports array
- writes entities for user, role, membership and organization
- generates migrations file for all entities
- moves the test folder back to the root of the project as it was before i moved it to the src directory. I did this because it was causing some hidden errors.
#### Manual Testing Steps
- `npm install`
- Go to `.env` or create one (follow .env-sample) and add your database the credentials 
- run `npm run typeorm:run` and check ormlogs.log and also open database to see tables created
- run `npm run typeorm:revert` to drop all the tables in the database
#### New environmental Variables?
In `.env`: 
```
# Developement Database
DEV_PG_PORT=<number>
DEV_PG_HOST=<string>
DEV_PG_USER=<string>
DEV_PG_PASSWORD=<string>
DEV_PG_DATABASE=<string>
# Test Database
TEST_PG_HOST=<string>
TEST_PG_PORT=<number>
TEST_PG_USER=<string>
TEST_PG_PASSWORD=<string>
TEST_PG_DATABASE=<string>
SENTY_DSN=<string>
JWT_SECRET=<string>
```
#### Relevant Trello Tickets
- [Ticket 1](some.com)

### CODE REVIEW

File: ormconfig.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    PORT: process.env.PROD_PG_PORT,
+    USER: process.env.PROD_PG_USER,
+    PASSWORD: process.env.PROD_PG_PASSWORD,
+    DATABASE: process.env.PROD_PG_DATABASE,
+  },
+};
+
+const config = options[process.env.NODE_ENV || 'development'];
+
+module.exports = {
```
Comment (chidioguejiofor) [inline ormconfig.js:30]: nice

File: src/organization/organization.entity.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity({name: 'Organization'})
+export class Organization {
```
Comment (chidioguejiofor) [inline src/organization/organization.entity.ts:4]: We would need a verified attirbute here

File: src/user/user.service.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import { InjectRepository } from '@nestjs/typeorm';
+
+
+@Injectable()
+export class UsersService {
+        constructor(
+          @InjectRepository(UserRepository) private usersRepository: UserRepository,
+        ) {}
+      
+        // findAll(): Promise<User[]> {
```
Comment (chidioguejiofor) [inline src/user/user.service.ts]: why are these comments here?

**Review by chidioguejiofor — COMMENTED** [review]
Dripped some comments

File: src/organization/organization.entity.ts (hunk 2 above)
Comment (Runo-saduwa) [reply src/organization/organization.entity.ts:4]: Okay

File: src/user/user.service.ts (hunk 3 above)
Comment (Runo-saduwa) [reply src/user/user.service.ts]: I was testing something and  i guess i forgot to remove them 
ill do remove them now

**Review by chidioguejiofor — APPROVED** [review]

**Pull request merged**